src/ApplicationBuilderExtensions.cs

```diff
@@ -1,9 +1,6 @@
-﻿using Microsoft.AspNetCore.Http;
-using Microsoft.Net.Http.Headers;
-using System;
-using WebEssentials.AspNetCore.OutputCaching;
+﻿using Microsoft.AspNetCore.Builder;
 
-namespace Microsoft.AspNetCore.Builder
```

Comment:
This namespace was chosen so that you don't need to include a custom namespace in your startup.cs. This is how most NuGet packages do this